runelite-client/src/main/java/net/runelite/client/plugins/cooking/FermentTimerOverlay.java

```diff
@@ -72,7 +72,12 @@ public Dimension render(Graphics2D graphics)
 
 		panelComponent.getChildren().clear();
 
-		if (isMakingWine() || Duration.between(session.getLastWineMakingAction(), Instant.now()).getSeconds() < INITIAL_TIME)
+		if (isMakingWine())
+		{
+			session.updateLastWineMakingAction();
```

Comment:
This should be updated in the plugin so it is not constantly updated once per render frame.

Reply:
Agreed. Just updated this in the latest commit.